ui/start_ui.py

```diff
@@ -837,7 +830,7 @@ def _init_ui(self):
                     base_models_list.append("specify other models")
                     base_model_dropdown = gr.Dropdown(
                         base_models_list,
-                        value=base_models_list[2],
+                        value=base_models_list[4],
```

Comment:
add comment for using constant, what is the value you are specifying here. Same below